.github/workflows/build-distributions.yml

```diff
@@ -0,0 +1,48 @@
+name: Build installable distrabutions
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+    strategy:
+      matrix:
+        os: [ macos-latest ] #windows-latest, ubuntu-latest,
+        runner: [
+          {
+            name: arelleCmdLine,
+            executor: arelleCmdLine.py
+          },
+          {
+            name: arelleGUI,
+            executor: arelleGUI.pyw
+          }
+        ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Python 3
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+      - run: pip install -r requirements.txt pyinstaller
+      - uses: actions/checkout@v3
+        with:
+          repository: Arelle/EdgarRenderer
+          path: arelle/plugin/EdgarRenderer
```

Comment:
I'm not sure if it gets pulled into the build or not, but add a cleanup step to rm the EdgarRenderer .git directory just to be sure?